graphql/mappers/profile.js

```diff
@@ -8,51 +7,113 @@ export async function getProfileContent() {
   const mappedProfile = {
     en: {
       pageName: response.data.schPagev1ByPath.item.scTitleEn,
-      heading: response.data.schPagev1ByPath.item.scTitleEn,
-      cards: response.data.schPagev1ByPath.item.scFragments
-        .find((element) => element.scId === 'profile-cards')
-        .scItems.map((fragment) => {
-          return {
-            id: fragment.scId,
-            title: fragment.scTitleEn,
-            lists: {
-              tasks: fragment.schTasks.map((list) => {
+      heading: response.data.schPagev1ByPath.item.scFragments.find(
+        (element) => element.scId === 'profile-intro'
+      ).scContentEn.json[0].content[0].value,
+      list: response.data.schPagev1ByPath.item.scFragments
+        .map((element) => {
+          if (
+            element.scId !== 'profile-cards' &&
```

Comment:
We could probably reduce this to check if this is one of the three fragments ei, cpp, oas-profile we're looking for.

Reply:
Should be good!